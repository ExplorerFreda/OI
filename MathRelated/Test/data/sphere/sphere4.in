6
3911.500000 7721.500000 4720.000000 15739.000000 7940.500000 9635.500000
9777.500000 17648.500000 1673.000000 455.500000 13253.000000 7554.500000
16233.000000 17099.500000 5153.500000 15857.500000 10692.000000 4403.500000
125.500000 14746.500000 13662.500000 1473.000000 15804.000000 9743.500000
5843.000000 1829.000000 16783.000000 10702.500000 9627.000000 11727.500000
3046.500000 13348.000000 17682.000000 5628.500000 12618.500000 4662.500000
3579.500000 7110.000000 6242.000000 11589.000000 17704.500000 1879.000000
