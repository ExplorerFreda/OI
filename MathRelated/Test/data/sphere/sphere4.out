11380.102 11604.168 10019.885 9141.296 16039.581 12460.412
